kotlin/android/app/src/main/java/dev/firezone/android/features/session/ui/SessionActivity.kt

```diff
@@ -101,7 +101,11 @@ internal class SessionActivity : AppCompatActivity() {
             object : TabLayout.OnTabSelectedListener {
                 override fun onTabSelected(tab: TabLayout.Tab) {
                     viewModel.tabSelected(tab.position)
-                    refreshList()
+
+                    refreshList {
+                        // TODO: we might want to remember the old position?
```

Comment:
I'm not sure if we care too much about this, the scroll to position 0 was done because the screen started scrolled down which caused the internet resource no to show some times.

Reply:
I reckon moving to Jetpack Compose would make these questions go away.